packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -95,6 +96,14 @@ export const makeZoeSeatAdminKit = (
 
     getCurrentAllocationJig: async () => currentAllocation,
     getAllocationNotifierJig: async () => notifier,
+
+    wasWantSatisfied: async () => {
+      return E.when(payoutPromiseKit.promise, () =>
+        Object.keys(proposal.want).every(kwd =>
+          AmountMath.isGTE(currentAllocation[kwd], proposal.want[kwd]),
+        ),
+      );
+    },
```

Comment:
Id prefer that you use `satisfiesWant`, which you'll need to export from `offerSafety.js`. That way you'll already be prepared for multiples, which does export it defined as

```js
export const satisfiesWant = (proposal, allocation) =>
  satisfiesInternal(proposal.want, allocation, 1n) >= 1n;
harden(satisfiesWant);
```

This also makes clear that this PR is necessary! If clients had tested this themselves, they also would have created an incompatibility with multiples!

Reply:
I see that it was also exported on master, so no worries about that.

Reply:
Actually, I remember we had a discussion about this, and agreed it was better for the public method to return `0` or `1` for now, rather than `false` and `true`, so that it could be more informative once we have multiples. How many times was the `want` satisfied?

Reply:
I've updated offerSafety to return 0 or 1, and added tests for `satisfiesWant`.